tools/src/bin/build-blame.rs

```diff
@@ -114,7 +114,7 @@ fn unmodified_lines(
 }
 
 fn blame_for_path(
-    file_movement: &HashMap<Oid, HashMap<Oid, PathBuf>>,
+    file_movement: &Option<HashMap<Oid, PathBuf>>,
```

Comment:
nit: Passing `Option<&HashMap<..>>` is a bit more idiomatic (you'd move the `as_ref()` to the caller.

Reply:
This particular instance gets removed later but I updated the other one that remains.